enos/enos-scenario-e2e-docker-base-with-gcp.hcl

```diff
@@ -0,0 +1,160 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: BUSL-1.1
+
+# For this scenario to work, add the following line to /etc/hosts
+# 127.0.0.1 localhost boundary
```

Comment:
This might just be copied from the other TF files, but is there any way we can make the scenario error with this error visible to the user instead of just adding a note here? I worry that this is sort of difficult to discover.

Reply:
I also had this issue when testing locally :sweat_smile:. It took me some time to figure out . We could probably add something when we authenticate and cannot reach boundary. An message that could suggest this is the issue. Maybe in this function: https://github.com/hashicorp/boundary/blob/ae0ca0c5aadcf87b9d82c13bbb31327b66993c70/testing/internal/e2e/boundary/authenticate.go#L56

@moduli thoughts?

Reply:
Sure. We could put a message there if `output := e2e.RunCommand(ctx, "boundary", options...)` returns an error. Just want to make sure the error states that it's also possible that the address is no longer valid.